### PR TITLE
Note about Adblockers in this repo

### PR DESCRIPTION
**Pull request by seuros**
Editing a PR/issue fail when an Adblocker is active

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,3 +95,7 @@ other news.
  - [AdsCommon ChangeLog](https://github.com/googleads/google-api-ads-ruby/blob/master/ads_common/ChangeLog)
  - [AdWords library ChangeLog](https://github.com/googleads/google-api-ads-ruby/blob/master/adwords_api/ChangeLog)
  - [DFP library ChangeLog](https://github.com/googleads/google-api-ads-ruby/blob/master/dfp_api/ChangeLog)
+
+### Note about this repo:
```
Comment (mcloonan) [inline README.md]: Can you use the same heading level as the rest of the file (##) and rename this section to "Contributing"

File: README.md [hunk 2]
```diff
@@ -95,3 +95,7 @@ other news.
...
+If you have an ad blocker active, you might experience issue while submitting an issue or pull request.
```
Comment (mcloonan) [inline README.md]: ...you might experience issues...

File: README.md [hunk 3]
```diff
@@ -95,3 +95,7 @@ other news.
  - [AdsCommon ChangeLog](https://github.com/googleads/google-api-ads-ruby/blob/master/ads_common/ChangeLog)
  - [AdWords library ChangeLog](https://github.com/googleads/google-api-ads-ruby/blob/master/adwords_api/ChangeLog)
  - [DFP library ChangeLog](https://github.com/googleads/google-api-ads-ruby/blob/master/dfp_api/ChangeLog)
+
+### Note about this repo:
+If you have an ad blocker active, you might experience issue while submitting an issue or pull request. 
+Please disable it for this url.
```
Comment (mcloonan) [inline README.md]: "You can still contribute by disabling the ad blocker for this project url."

**Comment by seuros** [comment]
@mcloonan fixed.

**Pull request merged**